sync-with-upstream/entrypoint.sh

```diff
@@ -11,6 +11,22 @@ DOWNSTREAM_BRANCH="$3"
 [[ -n "${UPSTREAM_BRANCH}" ]] || exit 1
 [[ -n "${DOWNSTREAM_BRANCH}" ]] || exit 1
 
+#
+# If there is a PR already then this is a no-op.
+#
+# An alternative design would be to pile up more commits
+# from the upstream repo if there exist any into the
+# existing PR instead of exiting here. (e.g. we could
+# check the hash of the upstream HEAD and see if that
+# hash is part of the existing PR, and if it's not we'd
+# force push into the PR). We decided not to go with
+# that design due to the simplicity of the current one.
+#
+if [[ $(hub pr list -s open -h "projects/sync-with-upstream/${DOWNSTREAM_BRANCH}" | wc -l) -ge 1 ]]; then
```

Comment:
Not that it really matters, but this could likely be done (IMO) more succinctly using `-n`; e.g.
```
if [[ -n "$(hub pr list -s open -h "projects/sync-with-upstream/${DOWNSTREAM_BRANCH}")" ]]; then
```